frontend/app/(defaultLayout)/Troubleshooting/[device]/page.tsx

```diff
@@ -4,13 +4,16 @@ import { Metadata } from 'next';
 import { headers } from 'next/headers';
 import { notFound } from 'next/navigation';
 import TroubleshootingProblems from './components/troubleshootingProblems';
-import { TroubleshootingProblemsApiData } from './hooks/useTroubleshootingProblemsProps';
+import {
+   TroubleshootingProblemsApiData,
+   TroubleshootingAnswersData,
+} from './hooks/useTroubleshootingProblemsProps';
 import {
    IFixitAPIClient,
    VarnishBypassHeader,
 } from '@ifixit/ifixit-api-client';
 
-export type AnswersProps = {
+export type AnswersProps = Omit<TroubleshootingAnswersData, 'answers'> & {
    allAnswersUrl: string;
    allAnswersCount: number;
 };
```

Comment:
We don't need the `& { ...` anymore because these are covered by the omit. [Here is a quick example](https://chat.openai.com/share/0e8e1207-bdf5-42d8-8827-f6cb2c3853d0) of how Omit works to declare a type.

```suggestion
export type AnswersProps = Omit<TroubleshootingAnswersData, 'answers'>;
```